### DISCUSSION TITLE
NEW LSM IPE merged in Kernel 6.12

### DISCUSSION THREAD

**Discussion by osevan** [netblue30/firejail#6480] [General]
https://docs.kernel.org/next/admin-guide/LSM/ipe.html

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=a430d95c5efa2b545d26a094eb5f624e36732af0
Feedbacks are welcome.

Maybe additional feature for Firejail.

**Comment by rusty-snake** [comment]
Who should it be integrated / support look like?
